lib/babe/epoch.go

```diff
@@ -138,6 +139,7 @@ func (b *Service) getEpochDataAndStartSlot(epoch uint64) (*epochData, uint64, er
 		authorities:    data.Authorities,
 		authorityIndex: idx,
 		threshold:      threshold,
+		secondary:      types.AllowedSlots(cfgData.SecondarySlots),
```

Comment:
I would suggest to rename `secondary` to `slotProduction` as this property enables/not enable the vrf/plain secondary block productions
```suggestion
		slotProduction:      types.AllowedSlots(cfgData.SecondarySlots),
```